### PR TITLE
feat(Rest) : Download license clearing spreadsheet.

### PR DESCRIPTION
**Pull request by keerthi-bl**
[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

closes : #2248 

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)

 #2248 

> * Did you add or update any new dependencies that are required for your change?
No

URL : http://localhost:8080/resource/api/reports?withlinkedreleases=false&module=projects&projectId=bf1570a235414de486eb1043d4b0c0a8&mailrequest=false

Sample respo : 

Excel should get downloaded.

> Have you implemented any additional tests?

### Checklist
Must:
- [ ] All related issues are referenced in commit messages and in PR

### CODE REVIEW

**Comment by nikkuma7** [comment]
Test successful.

image

image

File: rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ProjectSpecTest.java [hunk 1]
```diff
@@ -2051,6 +2051,53 @@ public void should_document_get_project_report_without_mail_req() throws Excepti
                         )));
     }
 
+    @Test
+    public void should_document_get_project_licenseclearing_report() throws Exception {
```
Comment (smrutis1) [inline rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ProjectSpecTest.java:2055]: Please update the method name as its not the clearing report but the spreadsheet export.

**Review by smrutis1 — CHANGES_REQUESTED** [review]
One thing I observed that the downloaded spreadsheet name from UI and Rest is different. Please make it consistent. Seems like the project name is not there in the filename.

**Review by smrutis1 — APPROVED** [review]
CLGTM

**Pull request merged**